src/DynamoCore/Engine/EngineController.cs

```diff
@@ -518,8 +513,18 @@ private void OnLibraryLoaded()
         /// </summary>
         private void LibraryLoaded(object sender, LibraryServices.LibraryLoadedEventArgs e)
         {
-            if(e.LibraryPaths.Any())
+            if (e.LibraryPaths.Any())
```

Comment:
so libraryPaths will include paths to custom nodes or only binaries?

Reply:
This is used for binaries only